src/components/TransactionPreview/ActionPreview.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useMemo, useState } from "react";
 import styled, { useTheme } from "styled-components";
 import { ethers } from "ethers";
 import { HiOutlinePencilAlt } from "react-icons/hi";
-import { BsClockHistory, BiCheck, IoClose, FaSignature } from "react-icons/all";
+import { BsClockHistory, BiCheck, IoClose, FaSignature , CgSandClock} from "react-icons/all";
```

Comment:
its better to import the icons from their respective files like `react-icons/fa` `react-icons/io` etc